.github/workflow_templates/dispatch-slash-command.yml

```diff
@@ -50,6 +52,12 @@ jobs:
               return core.setOutput('trigger_for_changelog', 'true');
             }
 
+            // only one for abort
+            if (isPR && (authorAssociation === 'OWNER' || authorAssociation === 'MEMBER' || authorAssociation === 'CONTRIBUTOR' || (isPrivate && authorAssociation === 'COLLABORATOR'))) {
+              core.notice(`Comment on pull request.`);
+              core.setOutput('trigger_for_pull_request', 'true');
+              return core.setOutput('issue_context', JSON.stringify(context.payload.issue));
```

Comment:
Is it needed to use output for passing payload.issue? I think it will not change during the workflow run and payload is accessible in each 'github-script' step.